Cargo.toml

```diff
@@ -37,7 +37,7 @@ profiler = ["alloc/profiler"]
 rate_limit = ["dep:rate_limit"]
 
 [workspace.dependencies]
-aws-sdk-s3 = "1.13"
+aws-sdk-s3 = "1.21.0"
```

Comment:
Can we remove the patch version number and use only `1.21`?

Reply:
Rust dependency resolution allows higher patch versions to be compatible with lower ones. This is setting a minimum version which was copy/pasted from crates.io and so should be OK to do. Setting to a lower version would be lying about which versions you've tested with as Cargo.lock is initially set to the latest version. More details [here](https://github.com/WalletConnect/WalletConnectRust/pull/62#issuecomment-1946463826)